### ISSUE TITLE
Best hyperparameters for text similarity search?

### ISSUE THREAD

**Issue by 0x01h** [epfml/sent2vec#96]
Hi,

I try to embed some sentences that have 5-20 number of tokens and have little syntactic differences. Also, I collected a corpus to train for that specific domain.

I think, `-loss ns` and selecting appropriate threshold of sampling `-t` to select "*negative hard mining samples*" are important.

What are the best hyperparameters to use these kind of case which focuses on text similarity?

**Comment by mpagli** [comment]
I'd use one of our pre-trained models for this. Try using the wiki or twitter model, this should give you already interesting results.
